pkg/kanctl/kanctl.go

```diff
@@ -39,6 +39,12 @@ var (
 	Verbose bool
 )
 
+type Clients struct {
+	KubeClient kubernetes.Interface
+	CrdClient  versioned.Interface
+	OsClient   osversioned.Interface
```

Comment:
s/OsClient/OSClient

following the Golang convention on initialism: https://github.com/golang/go/wiki/CodeReviewComments#initialisms